### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by mayahp**
Landing page has nav bar and search bar, which uses places api.
Screenshot 2020-06-26 at 10 47 23 AM - Display 1
Screenshot 2020-06-26 at 10 47 34 AM - Display 1

### CODE REVIEW

File: frontend/src/components/welcome.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+                	</Form>
+                </Jumbotron>
+     		</div>
+        );
+  	}
+
+    handleScriptLoad() { 
+    	// Declare Options For Autocomplete 
+
+        const types = ["university"];
```
Comment (Jkim-Hack) [inline frontend/src/components/welcome.js:45]: indent issue here

File: frontend/src/components/welcome.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+	}
+
+    handlePlaceSelect() {
+        const addressObject = this.autocomplete.getPlace();
+        const address = addressObject.address_components;
+
+        if (address) {
+            this.setState(
+                {
+                    city: address[0].long_name,
```
Comment (Jkim-Hack) [inline frontend/src/components/welcome.js:61]: indent issue here, should be only +2 whitespaces not +4

File: frontend/src/components/welcome.js [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    	this.state = {
+    		city: '',
+    		query: ''
+    	};
+
+	}
+
+  	render() {
+    	return (
+      	    <div>
```
Comment (Jkim-Hack) [inline frontend/src/components/welcome.js:22]: indent error here

File: frontend/src/components/welcome.js [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+    	    	<Script url="https://maps.googleapis.com/maps/api/js?key=KEY&libraries=places" onLoad={this.handleScriptLoad}/>
+                <Jumbotron>
+                	<h1>Welcome to MapIT!</h1>
+                	<Form>
+                		<Form.Group controlId="universityForm">
```
Comment (Jkim-Hack) [inline frontend/src/components/welcome.js:27]: make sure theres only +2 whitespace for every new line under the scope

File: frontend/src/components/welcome.js [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+class Search extends Component {
+  
+	// Define Constructor
+	constructor(props) {
+    	super(props);
+        this.handleScriptLoad = this.handleScriptLoad.bind(this);
+
+    	// Declare State
+    	this.state = {
+    		city: '',
```
Comment (Jkim-Hack) [inline frontend/src/components/welcome.js:14]: indent should be only +2 whitespace

File: frontend/src/components/welcome.js [hunk 6]
```diff
@@ -0,0 +1,69 @@
+import React, { Component } from 'react';
+import Script from 'react-load-script';
+import { Form, Button, Jumbotron } from 'react-bootstrap'
+
+class Search extends Component {
+  
+	// Define Constructor
+	constructor(props) {
```
Comment (Jkim-Hack) [inline frontend/src/components/welcome.js:8]: indent should be only +2 whitespace

File: frontend/src/components/welcome.js [hunk 7]
```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react';
+import Script from 'react-load-script';
+import { Form, Button, Jumbotron } from 'react-bootstrap'
+
+class Search extends Component {
+
```
Comment (tmicomyiza) [inline frontend/src/components/welcome.js:6]: remove extra space

File: frontend/src/components/welcome.js [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+  constructor(props) {
+    super(props);
+    this.handleScriptLoad = this.handleScriptLoad.bind(this);
+
+    // Declare State
+    this.state = {
+      city: '',
+      query: ''
+    };
+
```
Comment (tmicomyiza) [inline frontend/src/components/welcome.js:17]: extra space

File: frontend/src/components/welcome.js [hunk 9]
```diff
@@ -0,0 +1,63 @@
...
+  }
+
+  render() {
+    return ( 
+      <div>
+        <Script url = "https://maps.googleapis.com/maps/api/js?key=KEY&libraries=places" onLoad = {this.handleScriptLoad}/>
```
Comment (tmicomyiza) [inline frontend/src/components/welcome.js:23]: i think you can style it like this  in order to be consistent with other files.
<Script 
    url = "https://maps.googleapis.com/maps/api/js?key=KEY&libraries=places" 
      onLoad = {this.handleScriptLoad}/>

**Pull request merged**